exonum-java-binding/CHANGELOG.md

```diff
@@ -152,7 +152,7 @@ The release is based on Exonum 0.8.
 - Standard services may be enabled using specific `ejb_app_services.toml` file.
   It supports only `configuration` and `btc-anchoring` services at the moment.
 
-  To enable services put `ejb_app_services.toml` file into EJB App's directory
+  To enable services put `ejb_app_services.toml` file into Exonum Java app's directory
```

Comment:
This entry got me confused (it is applicable to the first release)

Reply:
Well, from my point of view, now "Exonum Java app's directory" corresponds to `exonum-java-binding/core/rust/exonum-java`. When the user will have provided packaged application, it will correspond to installation directory of `exonum-java` itself.